.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,39 @@
+---
+name: 🙋 Feature Request
+about: Want us to add something to FAST?
+title: "feat: add [what] to/in [where]"
+labels: 'status:triage'
+---
+
+<!---
+Thanks for filing an issue 😄 ! Before you submit, please read the following:
+
+Search open/closed issues before submitting. Someone may have requested the same feature before.
+-->
+
+# 🙋 Feature Request
+
+<!--- Provide a general summary of the feature here -->
+
+## 🤔 Expected Behavior
+
+<!--- Tell us how the feature should work -->
+
+## 😯 Current Behavior
+
+<!--- Explain how the feature would alter/enhance current behavior -->
+
+## 💁 Possible Solution
+
+<!--- Ideas how to implement this feature -->
```

Comment:
Perhaps, "What implementation or solution would be ideal for you?" instead of both sentences that ring fairly similar.